<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PdfMainWindow</class>
 <widget class="QDialog" name="PdfMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>513</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PDF Final</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-image: url(:/images/pdfback.jpg);
    background-repeat: no-repeat;
}

QWidget{
font: 11pt &quot;MS Shell Dlg 2&quot;;
}

QRadioButton{
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>1. Please select a PDF file</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>30</number>
        </property>
        <item>
         <widget class="QLineEdit" name="editPdfFile">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectPdf">
          <property name="text">
           <string>PDF</string>
          </property>
          <property name="icon">
           <iconset resource="bf.qrc">
            <normaloff>:/images/pdf.png</normaloff>:/images/pdf.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>2. Sepcify if the PDF has multiple columns</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>30</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rdSingleColumn">
          <property name="text">
           <string>Single column</string>
          </property>
          <property name="icon">
           <iconset resource="bf.qrc">
            <normaloff>:/images/singleColumn.png</normaloff>:/images/singleColumn.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>Multiple columns</string>
          </property>
          <property name="icon">
           <iconset resource="bf.qrc">
            <normaloff>:/images/multiColumns.png</normaloff>:/images/multiColumns.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>3. Specify if you want to export images from PDF</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>30</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rdTextOnly">
          <property name="text">
           <string>Text only</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_4">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Text and images</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color: rgb(108, 108, 108);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>* If the PDF is made up of images only, please find other tools to export.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>4. Chapter mode</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>30</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rdOneChapter">
          <property name="text">
           <string>One chapter one book</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdChapterPerPage">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>One chapter per page</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdChapterAuto">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Guess chapters</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnConvert">
       <property name="text">
        <string>Convert to SNB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRead">
       <property name="text">
        <string>Read SNB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpload">
       <property name="text">
        <string>Upload to Bambook</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 0);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>PDF Final has been integrated into Bambook Factory.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="bf.qrc"/>
 </resources>
 <connections/>
</ui>
